.gitignore

```diff
@@ -11,7 +11,7 @@
 
 # misc
 .DS_Store
-.env*
+.env
```

Comment:
A dangerous change. It's normal for devs to create sensitive files like .env.local

I prefer to keep as-is then `git add -f .env.example`